app/Services/Phoenix/Phoenix.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Rogue\Services\Phoenix;
+
+use DoSomething\Northstar\Common\RestApiClient;
+use GuzzleHttp\Cookie\CookieJar;
+use Cache;
+
+class Phoenix extends RestApiClient
+{
+    /**
+     * Create a new Phoenix API client.
+     */
+    public function __construct()
+    {
+        $url = config('services.phoenix.uri') . '/api/' . config('services.phoenix.version') . '/';
+
+        parent::__construct($url);
+    }
+
+    /**
+     * Returns a token for making authenticated requests to the Drupal API.
+     *
+     * @return array - Cookie & token for authenticated requests
+     */
+    private function authenticate()
+    {
+        $authentication = Cache::remember('drupal.authentication', 30, function () {
+            $payload = [
+                'username' => env('PHOENIX_USERNAME'),
```

Comment:
ah I see that makes sense! Thanks for the explanation for why to use this instead of just the value in the .env file! Just changed and will merge :)